### DISCUSSION TITLE
Any new news about free GitHub Foundations Certification?

### DISCUSSION THREAD

**Discussion by 0xSm0ky** [community/community#143422] [GitHub Education] [answered]

**Comment by davevad93** [comment, marked as answer]
Hi @znxcvbio ,

There was a bug and they had to disable the offer, technically they should have fixed it by October 31 but they haven't done it yet.  

![1](https://github.com/user-attachments/assets/896bb6b8-dc91-4251-bf40-1619f3ac23dd)

I suggest you to subscribe to the discussion #138834 so you can stay tuned, as soon as the free certification is available again they will announce it there.

**Comment by MuhammadOwais-SE** [comment]
I think I know which bug it had. I just try to give the exam yesterday.
During my git foundation exam. I had a terrible experience and concern that have I able to submit my exam or not.

During the whole exam. A notification of "NAVIDIA Container is probited" keeping showing me on the screen every 10 seconds. Even I end the task before the exam. A notification keeps popping up.

After giving the exam. When I pressed "end session" button so I could move to 3rd portion. Nothing happened and when I pressed again I receiving a error full of logs. I refresh the browser multiple times. Unforunately, nothing happened. I wait to time to finished. But the browser freeze.

I report to PSI OPS Helpdesk but they just assigned the ticket number and request to contact github. The agent(Aoron) said my progress will be saved on the cloud, just contact github company

ticket number: [#4553992](https://enterprise.githubsupport.com/tickets/4553992).
